PowerShellEditorServices.build.ps1

```diff
@@ -34,7 +34,7 @@ if ($PSVersionTable.PSEdition -ne "Core") {
 
 task SetupDotNet -Before Clean, Build, TestHost, TestServer, TestProtocol, TestPowerShellApi {
 
-    $requiredSdkVersion = "2.0.0"
```

Comment:
Note that 2.0 has already reached the end of its life a year ago!
https://dotnet.microsoft.com/download/dotnet-core